examples/pos-mainchain/config/default/genesis_assets.json

```diff
@@ -152,20 +107,19 @@
 													"type": "array",
 													"items": {
 														"dataType": "bytes",
-														"minLength": 32,
-														"maxLength": 32
+														"length": 32
```

Comment:
here as well, we need to use "minLength", "maxLength"

Reply:
fixed